syntax = "proto3";

message StartGameC2S {
    
}